lua/entities/gmod_wire_expression2/core/custom/cl_cfc_e2_lib.lua

```diff
@@ -0,0 +1,20 @@
+-- PvP Functions
+E2Helper.Descriptions["playerIsInPvp"]     = "Returns 1 if a player is in Pvp, and 0 otherwise."
+E2Helper.Descriptions["playerIsInBuild"]   = "Returns 1 if a player is in build, and 0 otherwise."
+E2Helper.Descriptions["getPvpStatus"]      = "Returns \"pvp\" if a player is in Pvp, and \"build\" if they are in build. If a player is invalid, or their status is unknown, an empty string is returned."
+E2Helper.Descriptions["getPvpers"]         = "Returns a table of all players currently in Pvp."
+E2Helper.Descriptions["getBuilders"]       = "Returns a table of all players currently in Build."
+
+-- Ping Functions
+E2Helper.Descriptions["averagePing"] = "Returns a floating-point number representing the average ping of players in the server."
+
+-- String Functions
+E2Helper.Descriptions["startsWith"] = "Returns 1 if the string on which it was called starts with start, and 0 otherwise."
+
+-- Array Functions
+E2Helper.Descriptions["shuffled"]   = "Returns a shuffled array."
+E2Helper.Descriptions["difference"] = "Returns an array filled with the differing elements between the calling and argument arrays."
+E2Helper.Descriptions["reversed"]   = "Returns a reversed array."
+E2Helper.Descriptions["sub(N, N)"]  = "Returns an array representing the subset between indexes n1 and n2, inclusively."
+E2Helper.Descriptions["sub(N)"]     = "Returns an array representing the subset between index n and the end of the array, inclusively."
```

Comment:
Will be testing the validity of the `sub(N)/sub(N, N)` entries tonight.

Reply:
Speak for yourself, and get back to me with your findings.

Reply:
jkjk see you there